CMakeLists.txt

```diff
@@ -279,9 +281,7 @@ set(LOVR_SRC
 )
 
 if(EMSCRIPTEN)
-  set(LOVR_HEADSET src/headset/webvr.c)
-else()
-  set(LOVR_HEADSET src/headset/openvr.c)
```

Comment:
Hm, shouldn't we actually avoid compiling openvr.c when building for WebVR?  It isn't used there.